internal/command/debug_notification_model.go

```diff
@@ -0,0 +1,35 @@
+package command
+
+import (
+	"github.com/caos/zitadel/internal/domain"
+	"github.com/caos/zitadel/internal/eventstore"
+	"github.com/caos/zitadel/internal/repository/settings"
+)
+
+type DebugNotificationWriteModel struct {
+	eventstore.WriteModel
+
+	Compact bool
+	State   domain.NotificationProviderState
+}
+
+func (wm *DebugNotificationWriteModel) Reduce() error {
+	for _, event := range wm.Events {
+		switch e := event.(type) {
+		case *settings.DebugNotificationProviderAddedEvent:
+			wm.Compact = e.Compact
+			wm.State = domain.NotificationProviderStateDisabled
```

Comment:
ah just found out, that right now it will create a separate event if the provider is enable on creation

can we talk about that... especially the difference in api methods and events (5 states vs 3 methods with enabled as field)